packages/flutter/lib/src/material/data_table.dart

```diff
@@ -64,6 +65,8 @@ class DataColumn {
   /// The label should not include the sort indicator.
   final Widget label;
 
+  final String? heading;
```

Comment:
Can you add some doc string explaining what this property does? It may help with reviewing